ddtrace/contrib/grpc/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""
+The Grpc integration will trace queries made using the grpc library.
+
+Grpc will be automatically instrumented with ``patch_all``, or when using
+the ``ddtrace-run`` command.
+Grpc is instrumented on import. To instrument Grpc manually use the
+``patch`` function.
+    import grpc
+    from ddtrace import patch
+    patch(grpc=True)
+
+    # use grpc like usual
+
+To configure the Grpc integration on an per-channel basis use the
+``Pin`` API::
+    import grpc
```

Comment:
RST needs a blank line here for this to work.

```suggestion

    import grpc
```